src/Tests/Libs/GObject-2.0.Tests/Records/ValueTest.cs

```diff
@@ -20,6 +20,13 @@ public void ValueFromDataShouldContainGivenData(object data)
         v.Extract().Should().Be(data);
     }
 
+    public void VariantFromDataShouldContainGivenData(object data)
```

Comment:
- Attribute `[TestMethod]` is missing.
- Parameter `data` in testdefinition is obsolete.